key-manager/src/main/scala/org/bitcoins/keymanager/WalletStorage.scala

```diff
@@ -304,6 +304,33 @@ object WalletStorage {
     }
   }
 
+  def changeAesPassword(
+      seedPath: Path,
+      oldPasswordOpt: Option[AesPassword],
+      newPasswordOpt: Option[AesPassword]): MnemonicState = {
+    decryptMnemonicFromDisk(seedPath, oldPasswordOpt) match {
+      case Left(err) => sys.error(err.toString)
+      case Right(decrypted) =>
+        val fileName = seedPath.getFileName.toString
+        val backup = seedPath.getParent.resolve(fileName + ".backup")
```

Comment:
Is there any way we can test this backup logic?

Reply:
No :/